packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/src/main/kotlin/org/hyperledger/cactus/plugin/ledger/connector/corda/server/impl/ApiPluginLedgerConnectorCordaServiceImpl.kt

```diff
@@ -79,6 +78,25 @@ class ApiPluginLedgerConnectorCordaServiceImpl(
         val returnValue = flowHandle.returnValue.get(timeoutMs, TimeUnit.MILLISECONDS)
         val id = flowHandle.id
 
+        // allow returnValue to be something different to SignedTransaction
+        var callOutput: kotlin.Any? = null
+        var transactionId: kotlin.String? = null
+
+        if (returnValue is SignedTransaction) {
+            logger.trace("returnValue is SignedTransaction - using returnValue.id.toString() ...");
+            transactionId = returnValue.id.toString();
+            // returnValueChecked = returnValue.id.toString();
```

Comment:
@elenaizaguirre Would something like this work here so that the callOutput can stay mandatory?

```kotlin
callOutput = returnValue.tx.outputs;
```

Reply:
@petermetz As we have discussed before, I am returning other values